.github/workflows/continous-integration-docker.yml

```
@@ -312,6 +318,12 @@ jobs:
       saves_to_disk: true
       disk_suffix: tip
       height_grep_text: 'current_height.*=.*Height'
+    # As the general workflow concurrency group just get matched in Pull Requests,
+    # it has no impact on this job, but we want to cancel running full syncs in
+    # `main` if a new PR gets merged
+    concurrency: 
+      group: ${{ github.workflow }}-${{ github.ref }}
+      cancel-in-progress: true
```

If we're having a busy week, we could keep on merging PRs, and no full syncs will ever complete, because they keep getting cancelled. Instead, it's better to let the first sync finish, then launch the latest sync.

```suggestion
    # We don't want to cancel running full syncs on `main` if a new PR gets merged,
    # because we might never finish a full sync during busy weeks. Instead, we let the
    # first sync complete, then queue the latest pending sync, cancelling any syncs in between.
    # (As the general workflow concurrency group just gets matched in Pull Requests,
    # it has no impact on this job.)
    concurrency: 
      group: ${{ github.workflow }}-${{ github.ref }}
      cancel-in-progress: true
```

---

Oops I forgot to switch cancel-in-progress to false